networkdb/networkdb_test.go

```diff
@@ -517,6 +527,56 @@ func TestNetworkDBNodeJoinLeaveIteration(t *testing.T) {
 		t.Fatalf("The networkNodes list has to have be 2 instead of %d - %v", len(dbs[1].networkNodes["network1"]), dbs[1].networkNodes["network1"])
 	}
 
-	dbs[0].Close()
-	dbs[1].Close()
+	closeNetworkDBInstances(dbs)
+}
+
+func TestNetworkDBGarbageCollection(t *testing.T) {
+	keysWriteDelete := 5
+	config := DefaultConfig()
+	config.reapEntryInterval = 30 * time.Second
```

Comment:
Just want to make sure, I saw in the `network.db` it has `reapEntryInterval = 30 * time.Minute`. Are we reducing it to 30 seconds in order to save time in the test?

Reply:
yep, this is only for test purposes and not to delay the unit test CI

Reply:
Got it. Thanks